src/complate/index.js

```diff
@@ -38,15 +38,15 @@ export function makeTransform(referenceDir) {
 
 export class Bundle extends _Bundle {
 	async renderString(code, filename, context) {
-		let { renderToString } = await load("rodunj/src/render", "complate extension");
+		let JSXRuntime = await load("complate-ast/dist/runtime", "complate-ast runtime");
 
 		// TODO: generate unique file name to avoid potential race condition for
 		//       concurrent access with identical sources?
 		let id = await this.virtualModule(filename, code);
 		code = await this.generate(id);
 
-		let sandbox = context ? { ...context } : {};
-		let segments = vm.runInNewContext(code, sandbox);
-		return renderToString(...segments);
+		let sandbox = { ...context, ...JSXRuntime };
```

Comment:
In https://github.com/FND/gorgeon/pull/1/#discussion_r366864209 you'd suggested this could be changed? I'm not sure what that would look like.

Reply:
See above, we can configure complate-ast to emit the required import statement. However, I'm completely unsure how this interacts with lazy loading. If you can tell me how the hypothetical generated code would look like, I could make it happen.